src/metatrain/cli/export.py

```diff
@@ -80,6 +80,9 @@ def export_model(model: Any, output: Union[Path, str] = "exported-model.pt") ->
         torch.jit.save(model, path)
     else:
         extensions_path = "extensions/"
-        logger.info(f"Exporting model to {path} and extensions to {extensions_path}")
+        logger.info(
+            f"Exporting model to `{path}` and extensions to `{extensions_path}`"
```

Comment:
I think I usally prefer quotes for path and back ticks for variables. But feel free

```suggestion
            f"Exporting model to {path!r} and extensions to {extensions_path!r}"
```

Reply:
Are you sure that `!r` isn't going to print some other stuff if path and/or extension_path are path objects?

Reply:
Ahh probably you are right. But then I would go for single ticks instead of backticks